Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	
+++ b/main.py	(date 1732379685858)
@@ -78,7 +78,7 @@
 source_listbox.grid(row=1, column=0, columnspan=3, padx=10, pady=5)
 
 
-tk.Label(root, text="Be careful to select the windows folder (Config/125a41ab....../Windows) !!!!").grid(row=3, column=0, padx=10, pady=5, sticky="w")
+tk.Label(root, text="Be careful to select only the profile folder !!!!").grid(row=3, column=0, padx=10, pady=5, sticky="w")
 tk.Label(root, text="Destination Folder:").grid(row=4, column=0, padx=10, pady=5, sticky="w")
 tk.Entry(root, textvariable=destination_folder_var, width=50).grid(row=4, column=1, padx=10, pady=5, sticky="w")
 tk.Button(root, text="Browse", command=browse_destination_folder).grid(row=4, column=2, padx=10, pady=5)
